docs/_replacements/docs/apis/fluid-framework.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "fluid-framework Package"
+kind: "Package"
+type: apis
+package: "fluid-framework"
+unscopedPackageName: "fluid-framework"
+area: "libraries"
+summary: "The fluid-framework package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package (for example, @<!-- -->fluidframework/azure-client or @<!-- -->fluidframework/tinylicious-client)."
```

Comment:
@tylerbutler Let me know if you have better idea here. Our original fluid-framework.md is shadowed by this file now, so getting `.Site.Pages` data out of it, did not seem as feasible option.

Reply:
Yeah, this is a fine solution I think.

Reply:
However, I happened to be testing with a more recent version of hugo, and in those, the table of packages doesn't render. I think the mount resolution has changed in recent versions, but a fix that seems to work across versions is to rename this file to be `_replacements\docs\apis\fluid-framework\_index.md`.

Reply:
Good find!